packages/orchestration/README.md

```diff
@@ -131,28 +130,53 @@ const response = await orchestrationClient.chatCompletion({
 });
 
 const responseContent = response.getContent();
+const finishReason = response.getFinishReason();
+const tokenUsage = response.getTokenUsage();
 ```
 
-`getContent()` is a convenience method that parses the response and returns the model's output as a string.
+You can use the following convenience methods for handling chat completion responses:
 
-To retrieve the `finish_reason` for stopping the chat completion request, use the convenience method `getFinishReason()`:
+- `getContent()`: parses the response and returns the model's output as a string.
+- `getFinishReason()`: retrieves the finish_reason, explaining why the chat completion request stopped.
+- `getTokenUsage()`: provides token usage details, namely `total_tokens`, `prompt_tokens`, and `completion_tokens`.
 
-```ts
-const finishReason = response.getFinishReason();
-```
+#### Structured Outputs
 
-Use the `getTokenUsage()` convenience method to retrieve the token usage details of the chat completion request:
+The `response_format` under `templating` guarantees that the model's output aligns with the JSON schemas specified by developers.
```

Comment:
```suggestion
Setting `response_format` to `json_schema` under `templating` guarantees that the model's output aligns with the JSON schemas specified by developers.
```

Since `response_format` can technically also be `text` and `json_object`

Reply:
Actually since you pointed this out, I will change this to:
The `response_format` under `templating` guarantees that the model's output aligns with the schema type specified by developers.

The example here shows json but other types are also allowed. Keeping this line generic then makes more sense to me.